packages/orpc/src/client/interceptors.ts

```diff
@@ -0,0 +1,33 @@
+import type { InterceptableOptions, Interceptor } from "@orpc/shared";
+import * as Sentry from "@sentry/core";
+
+const sentryInterceptor: Interceptor<
+  InterceptableOptions,
+  Promise<unknown>
+> = async function sentryInterceptor({ next, path, input }) {
+  return await Sentry.startSpan(
+    {
+      name: `orpc.${path.join("/")}`,
+      attributes: {
+        "rpc.system": "orpc",
+        "rpc.method": path.join("."),
+        "rpc.arguments": input ? JSON.stringify(input) : undefined,
```

Comment:
we'd make these configurable (maybe respect the PII setting?)

easiest way is to make these factory functions tho so you can pass options to the middleware/interceptor